.rubocop.yml

```diff
@@ -1,6 +1,48 @@
+inherit_from: .rubocop_todo.yml
+
 require: 
 - rubocop-rspec
 
-Metrics/BlockLength:
+########## Code Style
+
+# We are ok using single and double quotes interchangeably
```

Comment:
_Are_ we though? 😁 

I'd vote to move this into the todo pile but if I'm alone in this, feel free to ignore me!

Reply:
Yeah maybe we should indeed.

Reply:
That's quite a hot topic in Ruby land IIRC 🔥 🍿

Reply:
Moved it back into the todo rules list in e4edfda2caea356df27dfd9a8221b13c7ff2e30c

I'll probably actually enable it and apply autocorrect on it at some point, better be consistent even if the rule might sometimes feel nitpicky